catalyst_voices/packages/catalyst_voices_brands/lib/src/themes/extended_color_scheme.dart

```diff
@@ -0,0 +1,219 @@
+import 'package:flutter/material.dart';
+
+@immutable
+class ExtendedColorScheme extends ThemeExtension<ExtendedColorScheme> {
+  const ExtendedColorScheme({
+    required this.textPrimary,
+    required this.textOnPrimary,
+    required this.textOnPrimaryContainer,
+    required this.textDisabled,
+    required this.success,
+    required this.onSuccess,
+    required this.successContainer,
+    required this.onSuccessContainer,
+    required this.warning,
+    required this.onWarning,
+    required this.warningContainer,
+    required this.onWarningContainer,
+    required this.onSurfaceNeutral08,
+    required this.onSurfaceNeutral012,
+    required this.onSurfaceNeutral016,
+    required this.onSurfacePrimaryContainer,
+    required this.onSurfacePrimary08,
+    required this.onSurfacePrimary012,
+    required this.onSurfacePrimary016,
+    required this.onSurfaceSecondary08,
+    required this.onSurfaceSecondary012,
+    required this.onSurfaceSecondary016,
+    required this.onSurfaceError08,
+    required this.onSurfaceError012,
+    required this.onSurfaceError016,
+    required this.iconsForeground,
+    required this.iconsBackground,
+    required this.iconsDisabled,
+    required this.iconsPrimary,
+    required this.iconsSecondary,
+    required this.iconsSuccess,
+    required this.iconsWarning,
+    required this.iconsError,
+  });
+
+  final Color? textPrimary;
```

Comment:
In dart styleguide they put fields first, constructors below. Here it's a flutter's approach which I think we don't want to use (constructor first, fields below).